src/components/Cart/cartContext.js

```diff
@@ -0,0 +1,46 @@
+import { createContext, useReducer, useEffect, useState } from "react";
+const axios = require('axios');
+
+export const CartContext = createContext();
+
+export const CartProvider = ({ children }) => {
+  const [cart, setCart] = useState([])
+  const [state, dispatch] = useReducer(cartFunc, cart);
+
+  useEffect(() => {
+    axios.get('https://e-commerce-backend.harshporwal1.repl.co/cart')
+      .then(response => setCart(response.data))
+  })
```

Comment:
add dependency to useEffect

Reply:
This is having some issues, we will discuss